app/build.gradle

```diff
@@ -59,4 +59,23 @@ dependencies {
     testImplementation "org.mockito.kotlin:mockito-kotlin:3.2.0"
     // Optional -- Mockk framework
     testImplementation "io.mockk:mockk:1.4.1"
+    def room_version = "2.4.2"
+
+    implementation "androidx.room:room-runtime:$room_version"
+    annotationProcessor "androidx.room:room-compiler:$room_version"
+
+    // optional - RxJava2 support for Room
+    implementation "androidx.room:room-rxjava2:$room_version"
+
+    // optional - RxJava3 support for Room
+    implementation "androidx.room:room-rxjava3:$room_version"
+
+    // optional - Guava support for Room, including Optional and ListenableFuture
+    implementation "androidx.room:room-guava:$room_version"
+
+    // optional - Test helpers
+    testImplementation "androidx.room:room-testing:$room_version"
+
```

Comment:
Nie lepiej wpisać tam wersję, żeby później sam sobie to poprawił jak będzie nowsza?

Reply:
Jeżeli nie będzie pobierać jakiś niestabilnych wersji to możesz to zrobić ( ja nwm jak xd)

Reply:
Poprawiłem